include/sound/sof/topology.h

```diff
@@ -75,11 +75,20 @@ struct sof_ipc_comp {
 #define SOF_MEM_CAPS_CACHE			(1 << 6) /**< cacheable */
 #define SOF_MEM_CAPS_EXEC			(1 << 7) /**< executable */
 
+#define SOF_BUF_FLAG_OVERRUN_NO_XRUN	(1 << 0) /**< overrun will cause ring
```

Comment:
@akloniex My brain gives a compiler error on this. "xrun" is a short hand to talk about both "overruns" and "underruns", so having a flag with "overrun_no_xrun" does not make sense.

Maybe SOF_BUF_FLAG_OVERRUN_IGNORE ?

Reply:
I think it's "No XRUN trigger" or something. But yeah, this needs to be written in a clearer way so it doesn't become our secret language.